scripts/package-publish-config-tag.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -e
+
+VERSION=$(npx pkg-jq -r .version)
+
+if npx --package @chatie/semver semver-is-prod "$VERSION"; then
+  npx pkg-jq -i '.publishConfig.tag="latest"'
+  echo "production release: publicConfig.tag set to latest."
+else
+  npx pkg-jq -i '.publishConfig.tag="next"'
+  echo 'development release: publicConfig.tag set to next.'
```

Comment:
I would like to explain this script: we are following the prod/dev versioning convention same as the Linux kernel here.

Copy from Linux Kernel Version Numbering - http://www.linfo.org/kernel_version_numbering.html :

> The second number denotes the major revision of the kernel version. It was formerly the case that even numbers indicated a stable release, that is, one that was deemed fit for production use (i.e., use in a non-experimental environment), such as 1.2, 2.4 or 2.6. Likewise, odd numbers, such as 1.1 or 2.5, have historically represented development releases. They were for testing new features and device drivers until they became sufficiently stable to be included in a stable release. However, this has changed starting with the Linux 2.6.x series, and new feature development now takes place in the same revision number.

Minor version numbering rules:

1. even numbers, such as 0.8, 0.12, is for production use.
1. odd numbers, such as 0.11 or 0.13, is development releases.

I have a long version about this here: <https://github.com/wechaty/wechaty/wiki/FAQ#3-how-to-understand-the-wechaty-semantic-versioning>